src/fiatconnect/ReviewScreen.tsx

```diff
@@ -436,17 +435,17 @@ function TransactionDetails({
     })
   const { t } = useTranslation()
   let tokenDisplay: string
-  switch (normalizedQuote.getCryptoType()) {
-    case CiCoCurrency.cUSD:
+  switch (normalizedQuote.getTokenId()) {
```

Comment:
I know refactoring FC isn't in scope, but could we just use tokenInfo.name here? I think the only difference would be for celo which would be Celo instead of CELO which seems fine. It would help avoiding having more references to hardcoded token ids in code, which I think we should try to minimize. I am going to make a ticket to avoid hardcoding token ids / addresses in the wallet separately.